core/src/appleMain/kotlin/CoreBluetoothAdvertisement.kt

```diff
@@ -0,0 +1,12 @@
+package com.juul.kable
+
+import com.benasher44.uuid.Uuid
+import platform.Foundation.NSData
+import platform.Foundation.NSUUID
+
+public interface CoreBluetoothAdvertisement : Advertisement {
+    public val identifier: NSUUID
+    public fun serviceDataAsNSData(uuid: Uuid): NSData?
+    public val manufacturerDataAsNSData: NSData?
+    public fun manufacturerDataAsNSData(companyIdentifierCode: Int): NSData?
```

Comment:
Is there a good reason for the `AsNSData` suffix on these symbols?

Reply:
https://github.com/JuulLabs/kable/pull/479#discussion_r1187978361